india_compliance/gst_india/report/summary_of_itc_availed/summary_of_itc_availed.py

```
@@ -0,0 +1,279 @@
+# Copyright (c) 2025, Resilient Tech and contributors
+# For license information, please see license.txt
+
+from collections import defaultdict
+from enum import Enum
```

Ruff (0.11.9)

`enum.Enum` imported but unused

Remove unused import: `enum.Enum`

(F401)